wp-content/plugins/AB-Content-Alterer/ab-content-alterer.php

```diff
@@ -14,18 +146,17 @@ class AbAlternateContent {
   public function __construct() {
     add_action('the_content', [$this, 'textAlternate']);
   }
+
   function textAlternate($text) {
-    $textToAlter = [
-      'leadpages',
-      'unbounce'
-    ];
-    $alterText = 'Instapage';
+    $textToAlter = get_option('ab_alterer_option_group')['alterFrom'];

```

Comment:
After fresh plugin install get_option('ab_alterer_option_group') won't be set (no save action on plugin settings page), thus calling get_option('ab_alterer_option_group')['alterFrom'] will result in an error.

Check if 'alterFrom' exists in get_option('ab_alterer_option_group') table first.